src/windowsdesktop/src/sfx/Directory.Build.targets

```diff
@@ -48,4 +48,24 @@
 
   <Target Name="ReturnProductVersion" Returns="$(Version)" />
 
+  <Target Name="CreatePackageOverrides">
+    <PropertyGroup>
+      <PackageOverridesInputPath>$(MSBuildThisFileDirectory)PackageOverrides.txt</PackageOverridesInputPath>
+      <PackageOverridesOutputPath>$(BaseOutputPath)PackageOverrides.txt</PackageOverridesOutputPath>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <CreatePackageOverridesTemplateProperty Include="ProductVersion=$(Version)" />
```

Comment:
Will this version always be consistent with the version of the packages that were built in runtime?  I know you were talking to be about listing preview versions.  
If this version is not consistent, then make a comment about why that's intentional and a safe assumption to make.